src/helper_functions.jl

```diff
@@ -2,6 +2,7 @@ export get_dist_moment_ind
 export get_dist_moments_ind_range
 export get_moments_normalizing_factors
 export integrate_SimpsonEvenFast
+export rflatten
```

Comment:
Let's not export this, as it's not really a feature of Cloudy.jl, but rather an internal implementation detail.